iot_api/user_api/resources/endpoints.py

```diff
@@ -1072,192 +1073,188 @@ class Register(Resource):
-                msg = MIMEMultipart('alternative')
-                msg['Subject'] = f"{config.BRAND_NAME} Account Confirmation"
-                msg['From'] = email.utils.formataddr((config.SMTP_SENDER_NAME, config.SMTP_SENDER))
-                msg['To'] = user.email
-                part = MIMEText(render_template(
-                    'activation.html',
-                    brand_name=config.BRAND_NAME,
-                    full_url=full_url
-                    ), 'html')
-                msg.attach(part)
-                server = smtplib.SMTP(config.SMTP_HOST, config.SMTP_PORT)
-                server.ehlo()
-                server.starttls()
-                # stmplib docs recommend calling ehlo() before & after starttls()
-                server.ehlo()
-                server.login(config.SMTP_USERNAME, config.SMTP_PASSWORD)
-                server.sendmail(config.SMTP_SENDER, user.email, msg.as_string())
-                server.close()
-                LOG.debug("finished email sending")
-                new_account_activation.save_to_db()
+                    LOG.debug('init email sending')
+                    msg = MIMEMultipart('alternative')
+                    msg['Subject'] = f"{config.BRAND_NAME} Existing Account"
+                    msg['From'] = email.utils.formataddr((config.SMTP_SENDER_NAME, config.SMTP_SENDER))
+                    msg['To'] = email_without_space
+                    part = MIMEText(render_template(
+                        'existing_account.html',
+                        brand_name=config.BRAND_NAME,
+                        full_url=full_url
+                        ), 'html')
+                    msg.attach(part)
+                    server = smtplib.SMTP(config.SMTP_HOST, config.SMTP_PORT)
+                    # server.set_debuglevel(1)
+                    server.ehlo()
+                    server.starttls()
+                    # stmplib docs recommend calling ehlo() before & after starttls()
+                    server.ehlo()
+                    server.login(config.SMTP_USERNAME, config.SMTP_PASSWORD)
+                    server.sendmail(config.SMTP_SENDER, email_without_space, msg.as_string())
+                    server.close()
+                    LOG.debug("finished email sending")
 
-                return {
-                    "message": "An email was sent to the account provided"
-                }
-            elif config.SEND_EMAILS:
-                LOG.error("Activation mail not sent because there is not SMTP server configured.")
-                return internal("Something went wrong trying to send activation")
+                    return {
+                        "message": "An email was sent to the account provided"
+                    }
+                elif config.SEND_EMAILS:
+                    LOG.error("Existing account mail not sent because there is not SMTP server configured.")
+                    raise Error.Internal("Something went wrong trying to send mail")
```

Comment:
These two lines can be combined in something like:
`raise Error.Internal("Something went wrong trying to send mail: existing account mail not sent because there is not SMTP server configured.")`

Reply:
That would be cleaner. I will change it.